src/main/java/frc/robot/subsystems/LimelightSubsystem.java

```diff
@@ -45,20 +53,51 @@ public void periodic() {
       SmartDashboard.putNumber("target Id", table.getEntry("tid").getDouble(0.0));
```

Comment:
Is there a point to having all of these smart dashboard inputs, it is starting to crowd the dashboard